pubspec.yaml

```diff
@@ -1,17 +1,24 @@
 name: adhara_socket_io
 description: Websocket by socket.io for flutter by adhara, supports both iOS and Android
-version: 0.4.1
+version: 0.4.2
 author: Rohit R. Abbadi <rohit@infitio.com>
 homepage: https://github.com/infitio/flutter_socket_io
 
 environment:
-  sdk: ">=2.0.0-dev.68.0 <3.0.0"
+  sdk: ">=2.7.0 <3.0.0"
+  flutter: ">=1.10.0"
 
 dependencies:
   flutter:
     sdk: flutter
 
 flutter:
   plugin:
-    androidPackage: com.infitio.adharasocketio
-    pluginClass: AdharaSocketIoPlugin
+    platforms:
+      android:
+        package: com.infitio.adharasocketio
+        pluginClass: AdharaSocketIoPlugin
+#      ios:
+#        pluginClass: AdharasocketioPlugin
```

Comment:
I need some help with this section.

Reply:
@infitio could you help me with this?

Reply:
@Saifallak what is it?

Reply:
![image](https://user-images.githubusercontent.com/6053156/82742746-2fa63900-9d62-11ea-91b2-19ec1fc1a381.png)

this is making the pub knows the plugins support which platforms.
I'm not iOS expert, it just tells us to add the `pluginClass` here.
IDK what is the plugin class name in iOS,

but it's urgent to be added, because as what you can see in the screenshot above, the pub now treats this plugin is `Android` Only

Reply:
@tiholic ping

Reply:
Okay! I'll look into this ASAP

Reply:
Released an update

Reply:
great 👍